README.md

```diff
@@ -85,6 +85,12 @@ In some cases we rewrote popular GenAI models to be significantly faster in nati
 
 ### Training
 
+#### Float8
+
+[torchao.float8](torchao/float8) implements training recipes with the scaled float8 dtypes, as laid out in https://arxiv.org/abs/2209.05433.
```

Comment:
wanna mention a topline speedup marketing number?

Reply:
our last public number is from 2023H2, we plan to release new speedups in ~weeks but not ready yet.  Will add it here when it's posted.